lib/src/spotify_base.dart

```diff
@@ -12,37 +12,54 @@ abstract class SpotifyApiBase {
   bool _shouldWait = false;
   late FutureOr<oauth2.Client> _client;
   late Artists _artists;
+
   Artists get artists => _artists;
   late Albums _albums;
+
   Albums get albums => _albums;
   late Browse _browse;
+
   Browse get browse => _browse;
   late Tracks _tracks;
+
   Tracks get tracks => _tracks;
   late Playlists _playlists;
+
   Playlists get playlists => _playlists;
   late Episodes _episodes;
+
   Episodes get episodes => _episodes;
   late RecommendationsEndpoint _recommendations;
+
   RecommendationsEndpoint get recommendations => _recommendations;
   late Markets _markets;
+
   Markets get markets => _markets;
   late Users _users;
+
   Users get users => _users;
   late Search _search;
+
   Search get search => _search;
   late AudioFeatures _audioFeatures;
+
   AudioFeatures get audioFeatures => _audioFeatures;
   late AudioAnalysisEndpoint _audioAnalysis;
+
   AudioAnalysisEndpoint get audioAnalysis => _audioAnalysis;
   late Categories _categories;
+
   Categories get categories => _categories;
   late Me _me;
+
   Me get me => _me;
   late PlayerEndpoint _player;
+
   PlayerEndpoint get player => _player;
   late Shows _shows;
+
   Shows get shows => _shows;
+
```

Comment:
Please either remove the linebreaks or group the fields and getters for each endpoint instead of breaking them up.

Reply:
👍